### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,45 @@
...
+    This connector pulls data from Pulsar topics and persists data to Google Cloud Pub/Sub.
+
+    ![](docs/pubsub-sink.png)
+
+Currently, Google Cloud Pub/Sub connector versions (`x.y.z`) are based on Pulsar versions (`x.y.z`).
+
+| Google Cloud Pub/Sub connector version | Pulsar version | Doc |
+| :---------- | :------------------- | :------------- |
+[2.8.x](https://github.com/streamnative/pulsar-io-google-pubsub/releases/tag/v2.8.2.0)| [2.8.0](http://pulsar.apache.org/en/download/) | - [Google Cloud Pub/Sub source connector](TBD)<br><br>- [Google Cloud Pub/Sub sink connector](TBD) |
+[2.9.x](https://github.com/streamnative/pulsar-io-google-pubsub/releases/tag/v2.9.1.0-rc2)| [2.9.0](http://pulsar.apache.org/en/download/) | - [Google Cloud Pub/Sub source connector](TBD)<br><br>- [Google Cloud Pub/Sub sink connector](TBD) |
```
[the remark this answers is not in the document]
Comment (Huanli-Meng) [reply README.md:23]: The PR is in WIP status. When the hub doc is ready, i will update the URL.

**Comment by nodece** [comment]
@Huanli-Meng Could you fix the conflict?

**Comment by Huanli-Meng** [comment]
@nodece the links in the README is updated. Please help review it and feel free to approve it if no more comment. Thanks.

**Review by nodece — APPROVED** [review]
LGTM

**Pull request merged**